test/Microsoft.AspNet.Mvc.IntegrationTests/TestMvcOptions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Framework.OptionsModel;
+
+namespace Microsoft.AspNet.Mvc.IntegrationTests
+{
+    public class TestMvcOptions : IOptions<MvcOptions>
```

Comment:
I had that initially in TestCommon, however that meant a reference to Mvc.Core even in projects which do not refer it.
The case in point is the ModelBinding.Test. Adding this to TestCommon would mean updating ModelBinding.Test 's project.json to include Mvc.Core. Didnt want to get into that debate right now.